server/mail/index.js

```diff
@@ -65,7 +65,7 @@ const EmailService = {
     return email
       .render('../server/mail/emails/confirmation/html', brandedParams)
       .then(html => {
-        const subject = `Ilmoittautumisvahvistus: ${params.event.title}`;
+        const subject = `${params.edited ? 'Muokkaus' : 'Ilmoittautumis'}vahvistus: ${params.event.title}`;
```

Comment:
The only difference between the initial confirmation mail and edit confirm is the subject, as the mail content is well fitting for both cases